test/integration/routes.test.js

```diff
@@ -153,6 +154,11 @@ describe('routes', function () {
         expect(getItemResult.status).to.equal(200)
         expect(getItemResult.body.id).to.equal(lastToken.body.id)
         expect(getItemResult.body.metadata_keys).to.deep.equal(['testFile'])
+
+        const timestamp = getItemResult.body.timestamp
```

Comment:
Might be a good idea to use -> https://www.npmjs.com/package/mockdate mockdate, so you don't need -> .to.be.within(now - 6000, now). Following practises like this can make tests flakey and harder to debug (not this instance)

`.to.be.within(now - 6000, now)`

Reply:
I've generally achieved similar results using sinon fakeTimers. It's the same approach under the hood. If I remember rightly though mocking out the global Date functionality doesn't play well with the polkadot api but that may be wrong.